.gitignore

```diff
@@ -46,3 +46,5 @@ ctx.c
 
 # Per-user configuration
 .python-version
+
+expected/
```

Comment:
odd that its not in gitignore to begin with, its currently not in the repo at all.
Anyway, this can be grouped together with the other directories in this file instead of stuck at the bottom

Reply:
I've moved it to #Project-specific ignores